test/functional/test-polyfill-event.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {install} from '../../src/polyfills/event.js';
+
+describes.fakeWin('Event', {}, env => {
+
+  beforeEach(() => {
+    env.win.Event = Object.create(self.Event.prototype);
+    env.win.Event.prototype = self.Event.prototype;
```

Comment:
Yes, without this the `beforeEach` hook fails due to a TypeError originating from `event.js`. It tries to assign window.Event.prototype as the prototype of the newly created function, but throws an error because window.Event is undefined (recall that the prototype assignment happens before we assign the new function to window.Event). I believe these two lines effectively mimic IE 9+ as well, i.e. window.Event is an object, and not function.